src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -117,6 +126,19 @@ public async Task DetectApiChanges(string buildId, string artifactName, string o
             }
         }
 
+        private async Task<IssueComment> GetExistingCommentForPackage(string packageName, string repoOwner, string repoName, int pr)
+        {
+            var comments = await _githubClient.Issue.Comment.GetAllForIssue(repoOwner, repoName, pr);

```

Comment:
Just double checking, I know the github API pagination means that sometimes you don't get all issue comments in one request. Is this client successfully making sure it gets every comment?

Reply:
Good question. I don't see any pagination-based request for issue comments. Let me try by adding 30 additional dummy comments on my test PR and check.

Reply:
Verified that paging is not causing any issue for GetAllForIssue here.